featuretools/computational_backends/feature_set.py

```diff
@@ -204,9 +204,14 @@ def _dependent_uses_full_entity(self, feature):
 # These functions are used for sorting and grouping features
 
 
-def _get_use_previous(f):
+def _get_use_previous(f):  # TODO Sort and group features for DateOffset with two different temporal values
```

Comment:
Can you put an issue up about sorting for DateOffset / How does sorting work with use_previous?

Reply:
Created #746